routers/web/explore/repo.go

```diff
@@ -49,8 +50,6 @@ func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
 		orderBy = db.SearchOrderByNewest
 	case "oldest":
 		orderBy = db.SearchOrderByOldest
-	case "recentupdate":
```

Comment:
Why remove these two lines?

Reply:
Otherwise `onlyShowRelevant = setting.UI.OnlyShowRelevantRepos && !ctx.FormBool("no_filter")` would be duplicated code. Behavior is to always have this mechanism when recentupdate is chosen(by default or explicitly).